src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -38,7 +38,7 @@ public void SetGet_Roundtrips()
         {
             using (HttpClientHandler handler = CreateHttpClientHandler())
             {
-                var creds = new NetworkCredential("username", "password", "domain");
+                var creds = SecretsHelper.CreateCredentials("username", "password", "domain");
```

Comment:
```suggestion
                NetworkCredential creds = SecretsHelper.CreateCredentials("username", "password", "domain");
```

Reply:
Similarly for other occurrences. The moment you exchange ctor for a method call, it's not obvious what type the variable is :cry: